controllers/admin/AdminOrdersController.php

```diff
@@ -1705,6 +1705,11 @@ public function renderView()
         if (!Validate::isLoadedObject($order)) {
             $this->errors[] = $this->trans('The order cannot be found within your database.', array(), 'Admin.Orderscustomers.Notification');
```

Comment:
no return here?

Reply:
it would seem logical, but I don't want to change the current (expected?) behavior

Reply:
seems legit. But not sure was the expected behavior :roll_eyes:

Reply:
Probably not.. but I don't think there is a reported issue for now

Reply:
And anyway this page is going to be migrated